semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/InjectSemanticdbOptions.java

```diff
@@ -0,0 +1,28 @@
+package com.sourcegraph.semanticdb_javac;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class InjectSemanticdbOptions {
```

Comment:
Just for the sake of someone coming to read this in the future, it mightnt be a bad idea to briefly explain how this works. Its a fairly dense function :smile:

Reply:
Good catch! This is the hairiest part of the PR and it was definitely missing documentation. I was considering writing this logic in a bash script originally, but figured it would be easier to maintain as a small Java program.